addons/mail/tools/mail_validation.py

```diff
@@ -22,6 +22,6 @@ def mail_validate(email):
         global _flanker_lib_warning
         if not _flanker_lib_warning:
             _flanker_lib_warning = True
-            _logger.info("The `flanker` Python module is not installed,"
-                           "so email validation fallback to email_normalize. Use 'pip install flanker' to install it")
+            _logger.info("The (optional) `flanker` Python module is not installed,"
+                           "so email validation will fallback to email_normalize.")
```

Comment:
Remove two extra spaces at line 26 but that's just linting nitpicking

Not sure where I found this inspiring message cc @amigrave

Reply:
done ;)